tests/test_training.py

```diff
@@ -45,39 +45,72 @@ def data_dir():
     return DataDir("test_training")
 
 
+@pytest.fixture(params=["ru", "zh"])
+def trg_lang(request):
+    return request.param
+
+
+@pytest.fixture
+def config(trg_lang):
+    zh_config_path = os.path.abspath(os.path.join(FIXTURES_PATH, "config.pytest.enzh.yml"))
+    return zh_config_path if trg_lang == "zh" else None
+
+
 @pytest.fixture()
-def vocab(data_dir):
-    shutil.copyfile("tests/data/vocab.spm", data_dir.join("vocab.spm"))
+def vocab(data_dir, trg_lang):
+    output_path = data_dir.join("vocab.spm")
+    if trg_lang == "ru":
```

Comment:
Suggestion: It will help in debugging failures if there is a print statement on which vocab was used.